lib/supplejack_api/record_redis_queue.rb

```diff
@@ -29,43 +29,44 @@ def pop_record_ids(method = :index, batch_size = 1000)
       result.ids.value.uniq || []
     end
 
+    # Fetches records to be indexed
     def records_to_index
-      @records_to_index ||= ::Record.where(:id.in => self.pop_record_ids(:index)).to_a
-      @records_to_index.keep_if {|r| r.should_index? }
-      @records_to_index
+      records ||= SupplejackApi.config.record_class.where(:id.in => pop_record_ids(:index)).to_a
```

Comment:
Does this need to be memwised? records is local and will always be `nil` right?

Reply:
I'll have a look, it looks like the instance variable has been removed in the copy

Reply:
Yeah I actually don't see how this word work as it's not an instance variable haha. Good spotting!